scheds/rust/scx_chaos/src/bpf/main.bpf.c

```diff
@@ -43,6 +43,10 @@ const volatile u32 random_delays_freq_frac32 = 1; /* for veristat */
 const volatile u64 random_delays_min_ns = 1; /* for veristat */
 const volatile u64 random_delays_max_ns = 2; /* for veristat */
 
+const volatile u32 cpu_freq_frac32 = 1;
+const volatile u64 cpu_freq_min = 0;
+const volatile u64 cpu_freq_max = SCX_CPUPERF_ONE;
```

Comment:
nit: this kfunc takes a u32, maybe the same for consistency?

Reply:
good call